uploadNeuroDB/NeuroDB/objectBroker/ConfigOB.pm

```diff
@@ -143,6 +143,23 @@ my $getConfigSettingRef = sub {
     return $result->[0]->{'value'};
 };
 
+
+=head3 &$getBoolean($value)
```

Comment:
Nitpicking here: I would rename getBoolean into getBooleanRef to emphasize the fact that this variable is a _reference_ to a method.